### ISSUE TITLE
paymentID disapear

### ISSUE THREAD

**Issue by past2017** [turtlecoin/turtlecoin#1083]
### System Information

**Operating System**: 
ubuntu 16.04

### TurtleCoin Build Information

**Version:** 
0.28.0
**Binary Source:** 
https://github.com/turtlecoin/turtlecoin/releases/download/v0.28.0/turtlecoin-linux-v0.28.0.tar.gz
### Issue Report
when send turtlecoin by below API:
```
{
    "transfers":[
        {
            "amount":12,
            "address":"to"
        }
    ],
    "paymentId":"12345678",
    "changeAddress":"xx",
    "anonymity":3
}
```

**Binary Reported:** 

#### Expected Behavior
send successfully with right paytmentID

#### Current Behavior
send successfully but paymentID disappear

**Comment by brandonlehmann** [comment]
Is your payment ID exactly 32 hexadecimal characters?

Can you give a transaction hash example of one that didn’t work? 

You might want to migrate to wallet-api and turtle-service is EOL and will be EOS at v1.0.0.

**Comment by past2017** [comment]
> Is your payment ID exactly 32 hexadecimal characters?
> 
> Can you give a transaction hash example of one that didn’t work?
> 
> You might want to migrate to wallet-api and turtle-service is EOL and will be EOS at v1.0.0.

txid is https://explorer.turtlecoin.lol/transaction.html?hash=a8e6566cfee515de418150753c3479739b8e032a5c79b8fe25cc9f39a53c279d which i assigned it a paymentId 0000000000000000000000000000000000000000000000000000001865119858
yes, we are preparing to migrate to wallet-api and discard turtle-service, but by now we still use turtle-servie with version 0.28.0

**Comment by brandonlehmann** [comment]
Bug acknowledged. It appears adding the payment ID via turtle-service is somehow dropping it in the pool nonce field instead.

Raw transaction:

```010002025a049fd079eeb30d01019de54d12f24a6d3d502d12e0a73c59662879f6b145bf0a10699c2abb9ca2fb030280b51804c98828c2fe0201014167ac68cfff26e9bf2aeee170d372bdeb644fd1bcdb77bb010e49fd3c482d64065a021ec3de8094c56593f197b00b68dea69089bb063447c8b7f570194d0a611cc2bc64028e832840d1db5d23ff8e5ef5bdc70b5552d3a73ca41aa67ef4b4cdd9dbfec0f8840702d2e631c6af3f9833b23343072a8ae4419e1489e248244cc519827b59a468ebd6d83602b001c1306836b02e7eeb919a27e42feeaaaa37f3a306ac5d553d94e0172a97f490bf0502bdd5810039c586d52a05ab9250081c5eee004730693341a16fdf102b3187292ae0a71202d6d8409ba403dfae093b93a043a2d1a9969ff7c58578a0587646c14ade26b45844012273f52863d31862385635de51591235c6bfcd4a1f4c1b273f687249760c45260721000000000000000000000000000000000000000000000000000000001865119858cb679ead323f66611be7aea14ffafc61a1c0a6f2c21a952be065b8c4f627060ce4a45ce8077628dbeabc88cd4be44ae4c2d35739ede8a85fe84529df3999190c830b61fdcceea93bee98a72beed27bff39c934c048ea8428be209e44b0530b0e9bbfd72112272cb760012a15d8913e1c0184a5172f085c9d644c95d53e7e2e0c383f1854047d975fc5767210490cfdf5eaa13fff8d98f4869adcfd930470c50932c3ce21f2ee25483e562422f026d0b43d6947d5433afd4c7247e390e78d80022cec02bc803d5f81bc8fa64d828e14f22338a2c7c688f1d31f24f2dfa32a9c00d1eeb80ccf7874d9b0423636c2c06132c6314e92a9d24eb94a3dc2d16642b309fd96aabdc210be880b8019110ffdd6ffb3dd30dd3e300ff4ebad45ca3420d90111dcb7aac8865e51fb515005cb3e50e7a67637f88d886a59514d12332c8f7400e8b965a5eb648e552922dc1ec5dc0c1deab38a2e37eecc434ca90d4511dee20ba04faecddddf900293383640f0b9e610c665fa04d076d02ecb489d241681810a19d95b2b5980b1e3480fa06b08ea01b01e54b0d7041af37306f9d012e46e140fd73853af2bd52ac32fa55512677e0532244cf03183d957536fb0e7f2f22b36033c5a0fe1ae5884e6ca41332ff0dba5d520aa5efdea52b29e7e559cdfc06520056832d5a2c8341103dc3f7a5375ac28321d671be1587a75dd1143cd70fb851e01```

Extra:

```
012273f52863d31862385635de51591235c6bfcd4a1f4c1b273f687249760c45260721000000000000000000000000000000000000000000000000000000001865119858
```

01 (pub key): `2273f52863d31862385635de51591235c6bfcd4a1f4c1b273f687249760c4526`
07 (pool nonce): `000000000000000000000000000000000000000000000000000000001865119858`

This should be a `02` not `07` -- checking source now. Please standby.

**Comment by brandonlehmann** [comment]
Missed a reference to how that is generated by turtle-service as the pool nonce was back ported from the v1.0.0 branch. Working to correct now. Will package a release patch version.

**Comment by brandonlehmann** [comment]
Can you try building the `1083` [branch](https://github.com/turtlecoin/turtlecoin/tree/1083) and verify if that resolves your issue?

**Comment by brandonlehmann** [comment]
Bug resolved in #1084
